projects/ngneat/transloco-locale/README.md

```diff
@@ -0,0 +1,277 @@
+
+```typescript
+export class AppComponent {
+  constructor(private service: TranslocoLocaleService) {}
+
+  ngOnInit() {
+    this.service.setLocale('en-US');
+  }
+}
+```
+
+### Locale Format Options
+
+There are two types of formatting options, one for `date` and one for `number`.
+ 
+ 
+The formatted options could be declared in three levels 
+
+1. In the module's configuration (as mentioned above):
+
+```typescript
+import { TranslocoLocaleModule } from '@ngneat/transloco-locale';
+
+const globalFormatConfig = {
+  date: {
+    dateStyle: 'long',
+    timeStyle: 'long'
+  }
+}
+
+const esESFormatConfig = {
+  date: {
+    timeStyle: 'medium'
+  },
+  currency: {
+    minimumFractionDigits: 0
+  }
+}
+
+@NgModule({
+  imports: [TranslocoLocaleModule.init({
+    localeConfig: {
+      global: globalFormatConfig,
+      localeBased: {
+        'es-ES': esESFormatConfig
+      }
+    },
+  })],
+})
+export class AppModule {}
+```
+
+2. It could be set in the component's providers using `LOCALE_CONFIG` token:
+
+```typescript
+@Component({
+  selector: 'my-comp',
+  templateUrl: './my-comp.component.html',
+  providers: [
+    {
+      provide: LOCALE_CONFIG,
+      useValue: localeConfig
+    }
+  ]
+})
+export class MyComponent {}
+```
+
+3. We can pass it to each [pipe](#localization-pipes) in the HTML template:
+
+```html
+<span>
+  {{ date | translocoDate: { dateStyle: 'medium', timeStyle: 'medium' }}
+</span>
+
+<span>
+  {{ number | translocoDecimal: {useGrouping: false} }}
+</span>
+
+```
+
+Note the format option of the global, locale's format and the one's being passed in the template, will be merged. While the template is the stronger one and then the locale and the global.
+
+#### Number Format Options
+
+* `useGrouping `- Whether to use grouping separators, such as thousands separators or thousand/lakh/crore separators. Possible values are true and false; the (default is true).
+* `minimumIntegerDigits `- The minimum number of integer digits to use. Possible values are from 1 to 21 (default is 1).
+* `minimumFractionDigits `- The minimum number of fraction digits to use. Possible values are from 0 to 20 (default is 0).
+* `maximumFractionDigits `- The maximum number of fraction digits to use. Possible values are from 0 to 20 (default is 3).
+* `minimumSignificantDigits `- The minimum number of significant digits to use. Possible values are from 1 to 21 (default is 1).
+* `maximumSignificantDigits `- The maximum number of significant digits to use. Possible values are from 1 to 21 (default is 21).
+
+
+#### Date Format Options
+
+* `dateStyle` - The date formatting style.
+* `timeStyle` - The time formatting style.
+* `timeZone` - The time zone to use. The only value implementations must recognize is "UTC"; the default is the runtime's default time zone. Implementations may also recognize the time zone names of the [IANA](https://www.iana.org/time-zones) time zone database, such as "Asia/Shanghai", "Asia/Kolkata", "America/New_York".
```

Comment:
IANA time zones names do not work on IE11. How about referring to MDN via following link and inform people about this?

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleDateString#Browser_compatibility